pybedlite/overlap_detector.py

```diff
@@ -322,18 +322,38 @@ def overlaps_any(self, interval: Span) -> bool:
             True if and only if the given interval overlaps with any interval in this
             detector.
         """
-        tree = self._refname_to_tree.get(interval.refname)
+        tree = self._refname_to_tree.get(interval.refname, None)
```

Comment:
Doesn't `.get()` already return `None` as the default empty value?

Reply:
Yes. I tend to prefer explicit `None`s in this kind of use case and changed it without thinking much. I can revert if this is an issue.